### PR TITLE
Correct order of arguments for ValidatorStatusLogger

### PR DESCRIPTION
**Pull request by cemozerr**
## PR Description
Correct order of arguments for ValidatorStatusLogger

## Documentation

- [x] I thought about documentation and added the `documentation` label to this PR if updates are required.

### CODE REVIEW

File: validator/client/src/main/java/tech/pegasys/teku/validator/client/DefaultValidatorStatusLogger.java [hunk 1]
```diff
@@ -70,7 +70,7 @@ private void printValidatorStatusesOneByOne(
           Optional.ofNullable(validatorStatuses.get(publicKey));
       maybeValidatorStatus.ifPresentOrElse(
           validatorStatus ->
-              STATUS_LOG.validatorStatus(validatorStatus.name(), publicKey.toAbbreviatedString()),
+              STATUS_LOG.validatorStatus(publicKey.toAbbreviatedString(), validatorStatus.name()),
```
Comment (mbaxter) [inline validator/client/src/main/java/tech/pegasys/teku/validator/client/DefaultValidatorStatusLogger.java:73]: I think the problem is actually in `StatusLogger`: https://github.com/ConsenSys/teku/blob/f38689facd9ec15aa74065ad2462be3488d4e361/infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java#L255-L257

File: infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java [hunk 2]
```diff
@@ -252,8 +252,8 @@ public void unableToRetrieveValidatorStatusesFromBeaconNode() {
     log.error("Unable to retrieve validator statuses from BeaconNode.");
   }
 
-  public void validatorStatus(String validatorStatus, String publicKey) {
-    log.info("Validator {} status is {}.", validatorStatus, publicKey);
+  public void validatorStatus(String publicKey, String validatorStatus) {
+    log.info("Validator {} status is {}.", publicKey, validatorStatus);
```
Comment (mbaxter) [inline infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java:256]: Shouldn't this be?:
```suggestion
  public void validatorStatus(String validatorStatus, String publicKey) {
    log.info("Validator {} status is {}.", publicKey, validatorStatus);
```

Comment (cemozerr) [reply infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java:256]: Now that I've changed the public key to come first on the other side, this makes sense. Should I change both of them?

Comment (mbaxter) [reply infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java:256]: Oh, sorry - I was viewing a filtered changeset, so it looked like we only changed it here.

**Review by mbaxter — APPROVED** [review]

**Pull request merged**